### DISCUSSION TITLE
Loss of the x2 bonus XP following the recruitment of a friend after only 4 days playing

### DISCUSSION THREAD

**Discussion by Daliss** [chromiecraft/chromiecraft#2818] [General]
Hello I lost my x2 xp bonus on two of my characters and without any apparent reason after only 4 days of play
The characters in question are Safy and Maggy could you restore this feature if you play?

Thank you by Advance

**Comment by 55Honey** [comment]
Hello Daliss,
Thanks for your question.

You've been recruited by a player on the same IP adress. Which is not allowed according to the RAF Rules.
Your recruitment was removed automatically and can not be re-instated in any way. It's in the boldest possible letters: https://www.chromiecraft.com/recruit-a-friend-is-available-now/

**Comment by rsepierre** [comment]
@Daliss 
Hello Daliss, yes the recruit a friend feature is not instantly reactive to prevent people from abusing it. You might use it for multiple days without loosing your xp. 

There is still many software solutions to exploits the Refer-a-friend feature without installing specialized programs. 

I have reported them under this bug-tracker post : https://github.com/chromiecraft/chromiecraft/issues/2823

Because @55Honey said it is not considered to be an exploit. I can share with you the software method I use to gain an in-game advantage over people who wouldn't use my software method.
![image](https://user-images.githubusercontent.com/6746875/149961700-31b8fdb2-00e9-4128-9d09-267d0d87d33a.png)

Again, 55honey said it is not an exploit and closed the ticket, so you should be good to go yourself :

The technique to abuse the refer a friend feature and self-refer yourself is very straightforward and you simply need a phone.

![image](https://user-images.githubusercontent.com/6746875/149962059-b5303e85-136e-4c65-9696-a4d0a561e754.png)

__**Step by step exploit guide :**__
- Enable Connection sharing on your phone (mine is called "iPierre")
- Disconnect your computer from your local connection and connect to your phone's network

- For safety and to avoid being detected (even though I think it doesn't matter) you should clear your browser's cache and cookies and wow cache folder
- Create an account on [chromiecrafthromiecraft](https://www.chromiecraft.com/) which you will only use when connected to your phone.
- Log in the game, create a caracter
- Type .raf to get your referal code and save it somewhere you can find later
- Log out and disconnect from your phone network
- Connect back to your home internet with your computer 
- For safety and to avoid being detected (I still think it doesn't matter but it's for best measure) you should clear your browser's cache and cookies and wow cache folder
- Create the final account you wish to game on and use the referal code you saved earlier
- never log back in the first account (or use the phone connection again)
- enjoy 200% xp on your fresh account

I did try to report it as a bug and an exploit a bring out every problem it brings for no reason but it has been said it is not a bug nor an exploit so everyone should be able to use this technique and talk about it freely without expecting any retribution from the devs as we have the green light.
Now there are many other ways to use that feature without actually referring a friend :
- use a VPN
- ask for a referal code in /world (works even if it's not your friend)
- many others (but why bother going into more sophisticated techniques when the simpler ones works so well)

I actually do believe it would be simpler to freely give the 200% boost to everyone and keep the friend focused features for the RAF
But this point was not discussed by the admins.
https://github.com/chromiecraft/chromiecraft/issues/2823

**Reply by Amrx2** [reply]
Alternatively, you can save on all of these steps and just use anybody's code that they freely state in either the discord (its even in their names sometimes) or from world chat.

**Reply by rsepierre** [reply]
> Alternatively, you can save on all of these steps and just use anybody's code that they freely state in either the discord (its even in their names sometimes) or from world chat.

I stated this just after "use a VPN" in my section "many other ways to use that feature without actually refering a friend".

I understand that mods and devs don't care about the feature being abused and exploited.
I'm just stating that it is and that they don't care and maybe it's ok ?

But in that context, my question is then :
- Why hunt down people like @Daliss who self-refered instead of typing in /world "can i get code plz". (now he has to delete his account and start fresh. but he might also leave the server for good because his time was wasted).
- Why not make the xp boost accessible by other means ?
- Why make the feature unaccessible after 7 days ?

I don't think changing some of aspects of this would remove any marketability for chromiecraft (which is the goal of the refer-a-friend feature)
But I think it would smooth out the experience for edgecases like @Daliss who evidently want's to play on the server, but didn't know it was preferable to pretend having a friend instead of faking having a friend (ask on /6 world instead of self-refering without being careful to hide your tracks).
It would also enable one-home family referals (husband and wife, brothers and sisters) living under the same roof
Locking people out of an in-game advantage is always going to cause discussions and drama.

**Reply by 55Honey** [reply]
Dear @rsepierre,
you've been repeating the same over and over on various platforms. Our discord, spamming /world for hours, you've made your own discussion and you've hijacked this question to over and over repeat the same things. You've been told how to work around certain limitations and you've been told why they exist and why we allow workarounds. 

Once again:

Thank you for sharing your concerns and suggestions. We've heard them now many times today. But RAF is working as intended as it is. And further spamming is not going to change that answer.

**Reply by rsepierre** [reply]
I'm trying to help you make the feature problem and drama-free. I was answering to Amrx's reaction to this ongoing topic (less than a day old). I don't think I'm highjacking any discussion as Daliss and I have been affected by how the feature is currently implemented and I try to give him most accurate information on the current state of the feature and how to work woth it's limitations. I have not been told "why they (the limitations) exists and why we allow workarounds ". I have only been told what is the current state.
 And while you allow them, there still is an automated process hindering players experience such as daliss's and myself
